xdc-plugin/tests/counter_golden.json

```diff
@@ -0,0 +1,33 @@
+{
+  "OBUF_6": {
+    "DRIVE": "I12",
+    "IOSTANDARD": "LVCMOS33",
+    "SLEW": "SLOW"
+  },
+  "OBUF_7": {
+    "IN_TERM": "UNTUNED_SPLIT_40",
```

Comment:
Can you explain this one?  Based on my reading of the verilog (https://github.com/SymbiFlow/yosys-symbiflow-plugins/blob/bb93b7368f885447ee690c44383e4aa13aa0840a/xdc-plugin/tests/counter.v#L26), this is wrong?

Reply:
I figured it out, the XDC is overridding the verilog.

Reply:
Sure, in the [xdc file](https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/4/files#diff-08bea87e5e93e125875e7fa0dcc142e4R3) the IOSTANDARD gets overriden by setting the IN_TERM, SLEW and IOSTANDARD on led[0] which traverses through two intenal wires to OBUF_7.